Gave it a try and got this error in configurePhase:

```
util-linux> configure: WARNING: non-linux system; not building exch
util-linux> configure: WARNING: non-linux system; not building fincore
util-linux> configure: WARNING: non-linux system; not building fsfreeze
util-linux> configure: WARNING: non-linux system; not building blkdiscard
util-linux> configure: WARNING: non-linux system; not building blkzone
util-linux> configure: WARNING: non-linux system; not building blkpr
util-linux> configure: WARNING: non-linux system; not building ldattach
util-linux> configure: WARNING: non-linux system; not building rtcwake
util-linux> configure: WARNING: non-linux system; not building setarch
util-linux> configure: WARNING: openpty function (libutil) not found; not building script
util-linux> configure: WARNING: openpty function (libutil) not found; not building scriptlive
util-linux> configure: error: bits selected, but required cpu_set_t type not available
```

--- a/pkgs/by-name/ut/util-linux/package.nix
+++ b/pkgs/by-name/ut/util-linux/package.nix
@@ -3,6 +3,8 @@
   stdenv,
   fetchurl,
   pkg-config,
+  autoconf,
+  automake116x,
   zlib,
   shadow,
   capabilitiesSupport ? stdenv.hostPlatform.isLinux,
@@ -41,11 +43,19 @@ stdenv.mkDerivation rec {
     hash = "sha256-ge6Ts8/f6318QJDO3rode7zpFB/QtQG2hrP+R13cpMY=";
   };
 
-  patches = [
-    ./rtcwake-search-PATH-for-shutdown.patch
-    # https://github.com/util-linux/util-linux/pull/3013
-    ./fix-darwin-build.patch
-  ];
+  patches =
+    [
+      ./rtcwake-search-PATH-for-shutdown.patch
+      # https://github.com/util-linux/util-linux/pull/3013
+      ./fix-darwin-build.patch
+    ]
+    ++ lib.optionals (!stdenv.hostPlatform.isLinux) [
+      (fetchurl {
+        name = "bits-only-build-when-cpu_set_t-is-available.patch";
+        url = "https://lore.kernel.org/util-linux/20250501075806.88759-1-hi@alyssa.is/raw";
+        hash = "sha256-G7Cdv8636wJEjgt9am7PaDI8bpSF8sO9bFWEIiAL25A=";
+      })
+    ];
 
   # We separate some of the utilities into their own outputs. This
   # allows putting together smaller systems depending on only part of
@@ -137,6 +147,10 @@ stdenv.mkDerivation rec {
     [
       pkg-config
       installShellFiles
+    ]
+    ++ lib.optionals (!stdenv.hostPlatform.isLinux) [
+      autoconf
+      automake116x
     ]
     ++ lib.optionals translateManpages [ po4a ]
     ++ lib.optionals (cryptsetupSupport == "dlopen") [ cryptsetup ];

